scripts/airtable-access.R

```diff
@@ -0,0 +1,15 @@
+has_airtable_access <- function() {
```

Comment:
I'm not sure what the vision is of when and where this will be called, but using something like `if ("params" %in% ls())` should usually be done within a specific environment. A function like this would generally be written,
``` r
has_airtable_access <- function(env = .GlobalEnv) {
  ...
  if ("params" %in% ls(envir = env)) {
    ...
  }
}
```